geode-apis-compatible-with-redis/src/main/resources/org/apache/geode/redis/internal/sanctioned-geode-apis-compatible-with-redis-serializables.txt

```diff
@@ -1,8 +1,8 @@
 org/apache/geode/redis/internal/ParameterRequirements/RedisParametersMismatchException,true,-643700717871858072
 org/apache/geode/redis/internal/RedisCommandSupportLevel,false
 org/apache/geode/redis/internal/RedisCommandType,false,deferredParameterRequirements:org/apache/geode/redis/internal/ParameterRequirements/ParameterRequirements,executor:org/apache/geode/redis/internal/executor/Executor,parameterRequirements:org/apache/geode/redis/internal/ParameterRequirements/ParameterRequirements,supportLevel:org/apache/geode/redis/internal/RedisCommandSupportLevel
-org/apache/geode/redis/internal/cluster/BucketInfoRetrievalFunction,false,hostAddress:java/lang/String,redisPort:int
-org/apache/geode/redis/internal/cluster/BucketInfoRetrievalFunction$MemberBuckets,false,hostAddress:java/lang/String,memberId:java/lang/String,port:int,primaryBucketIds:java/util/Set
+org/apache/geode/redis/internal/cluster/RedisMemberInfoRetrievalFunction,true,2207969011229079993,hostAddress:java/lang/String,redisPort:int
```

Comment:
I think this function should really be in the excludedClasses instead. It's not sent over the wire.

Reply:
Right - done.